eks/managing_user_access.md

```diff
@@ -0,0 +1,107 @@
+# Managing Access to your Kubernetes cluster
+The following document assumes you have used the terraform scripts found in this repository to create a Kubernetes cluster for your CircleCI server installation. Below we go through our recommended approach to providing and managing user access.
+
+
+## Adding Cluster Administrators
+Cluster admins are first registered via appending their IPs and IAM user details to the appropriate lists in your `terraform.tfvars` file.
+
+example:
+```
+allowed_cidr_blocks = ["user_ip/32","user1_ip/32"]
+k8s_administrators = [
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user ARN>"
+      username = "<username>"
+  },
+  {
+      groups   = ["system:masters"]
+      userarn  = "<user1 ARN>"
+      username = "<user1name>"
+  }
+}
+```
+
+Upon `terrafrom apply`, these fields will whitelist your IPs, and add the list of users will be added to the system:masters group for full access to all resources in the cluster.
+
+### Updating the Cluster admin list
+If you wish to add/remove admin users from your existing cluster, you only need to update the values in your `terraform.tfvars` and run `terrafrom apply`.
+
+## Adding Users with Limited Resource Access
+You may not wish for each user to have complete access of to all cluster resources. To more finely tune user access we will make use kubernetes role based access control ([RBAC]).
+In the following example we will create a role which will have limited access to a `develop` namespace and then map it to an IAM user. You will need to be a cluster administrator as detailed above to proceed with the following steps.
+
+We'll first we'll need to add the user's ARN to the aws-auth configmap in the kube-system namespace.
```

Comment:
Garbled beginning of sentence. Also stay consistent with (not) using contractions – I personally lean towards not using them in written language

Reply:
another good catch :)